### ISSUE TITLE
Indentation is not preserved in array literal

### ISSUE THREAD

**Issue by jnm2** [paul1956/CSharpToVB#14]
Source lines: https://github.com/dotnet/roslyn/blob/c0a1d1ec94efc2b4fcc2334580c84741e10e45ef/src/EditorFeatures/CSharpTest/SimplifyInterpolation/SimplifyInterpolationTests.cs#L40-L42

Expected:

```vb
                Assert.Equal(
                    {
                        ("IDE0071", DiagnosticSeverity.Info),
                        ("IDE0071WithoutSuggestion", DiagnosticSeverity.Hidden),
                        ("IDE0071WithoutSuggestion", DiagnosticSeverity.Hidden)
```

Expected (alternative):

```vb
                Assert.Equal({
                    ("IDE0071", DiagnosticSeverity.Info),
                    ("IDE0071WithoutSuggestion", DiagnosticSeverity.Hidden),
                    ("IDE0071WithoutSuggestion", DiagnosticSeverity.Hidden)
```

Actual:

```vb
            Assert.Equal( _
                { _
("IDE0071", DiagnosticSeverity.Info),
("IDE0071WithoutSuggestion", DiagnosticSeverity.Hidden),
("IDE0071WithoutSuggestion", DiagnosticSeverity.Hidden)},
```

**Comment by paul1956** [comment]
This is the VB Formatter, after an explicit _ I have opened a bug Roslyn/#36891 that Sam Harwell as taking the lead on.

**Comment by paul1956** [comment]
Fixed in application 4.3.0.2 converter 3.3.0.2
